app/src/main/report/components/ReportsList/Filter/ReportsListFilterDate.tsx

```diff
@@ -1,53 +0,0 @@
-import * as React from "react";
-
-import {ReportsFilterFields} from "../../../actionTypes/ReportsActionsTypes";
-import {DatePicker} from "../../../../shared/components/DatePicker/DatePicker";
-import {CalendarIcon} from "../Calendar";
-
-interface ReportsListFilterProps {
-    filterData: ReportsFilterFields;
```

Comment:
The key difference between this and the new component is this `filterData` property, which tied it quite closely to the reports redux implementation, has been replaced with a `value` property that just has a start and end date.

Reply:
See new props interface: https://github.com/vimc/montagu-webapps/pull/232/files#diff-fccc52c233bf2632d2635c950bf8e88dR5